Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageView.swift

```diff
@@ -275,30 +276,16 @@ public struct LinkDetectionTextView: View {
     
     public var body: some View {
         Group {
-            if let displayedText {
-                Text(displayedText)
-            } else {
-                Text(message.adjustedText)
-            }
+            Text(displayText)
         }
         .foregroundColor(textColor(for: message))
         .font(fonts.body)
         .tint(tintColor)
-        .onAppear {
-            displayedText = attributedString(for: message)
-        }
-        .onChange(of: message, perform: { updated in
-            displayedText = attributedString(for: updated)
-        })
```

Comment:
I will probably need some help here CC @martinmitrevski  @laevandus. This was overriding the view model content always, and was causing some trouble to me. Why is this needed?